docs/api-testing/composer.md

```diff
@@ -2,82 +2,601 @@
+
+<img src={useBaseUrl('/img/api-testing/test-create-new-test.png')} alt="New Test window" width="400"/>
 
-#### Add Assertion Component
-11. Click the **Add Component** button (**+** icon).<br/><img src={useBaseUrl('img/api-fortress/2021/01/addComponent.png')} alt="Add Request Component" width="400" />
-12. Select the **Assert Exists** assertion component.<br/><img src={useBaseUrl('img/api-fortress/2021/01/assertExists.png')} alt="Assert Exists Component"/>
-13. Edit the following details:
-    * **Expression**: `payload.downloads` - This expression checks for the field `"downloads"` in the `json` response body.<br/><img src={useBaseUrl('img/api-fortress/2021/01/assertDetails.png')} alt="Assert Exists Details"/>
-    * Leave the rest of the fields blank and click **Confirm changes** (checkmark icon).
+5. On the **Compose** tab, make sure the **Input** and **Visual** views are toggled on.
 
-:::tip
-For detailed information, see [Assertion Components](/api-testing/composer/assertion-components/).
+<img src={useBaseUrl('/img/api-testing/input-visual-on.png')} alt="Input and Visual settings on" width="500"/>
+
+:::note
+You can use either the **Visual** composer (guides you through building components, with no coding required) or the **Code** composer (requires you to write code from scratch). For this guide, we're using **Visual**.
 :::
 
-### Run Test
-14. Before you run the test, select the **Save** icon.<img src={useBaseUrl('img/api-fortress/2021/01/save.png')} alt="save icon"/> at the top of the Composer.
-15. Then select the **Run** icon <img src={useBaseUrl('img/api-fortress/2021/01/run.png')} alt="run test icon"/> directly next to the Save icon.<img src={useBaseUrl('img/api-fortress/2021/01/composerToolbar.png')} alt="Test Composer Tool Bar"/>
+For more information, see [Input Sets](/api-testing/composer/#input-sets) and [Visual View and Code View](/api-testing/composer/#visual-view-and-code-view).
+
+### Edit a Test
+To edit a test at any time, on the **Projects** page, on the **Tests** tab, hover over a test name and then click **Edit Test**.
+
+<img src={useBaseUrl('/img/api-testing/edit-test-nav.png')} alt="Navigating to the test editor" width="300"/>
+
+### Add Test Components
+When test components are combined, they act as our test logic. There are three component types available in API Testing:
+  * [I/O Request Test Components](/api-testing/composer/io-components)
+  * [Assertion Test Components](/api-testing/composer/assertion-components/)
+  * [Logical Test Components](/api-testing/composer/logical-components/)
+
+#### Add an I/O Request Test Component
+To create a simple `GET` request and validate that response is correct:
+  1. In API Testing, on the **Compose** page, click the **Add component** button.
+
+  <img src={useBaseUrl('/img/api-testing/add-component-nav.png')} alt="Navigating to the Add component screen" width="600"/>
```

Comment:
@LoryApiFortress If you want to have a discussion offline about screenshot standards and guidelines, please set something up. Otherwise, I ask that you trust me to do my job.